papermill/tests/notebooks/read_check.ipynb

```diff
@@ -0,0 +1,44 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(\"NB_CWD\", os.getcwd())\n",
+    "assert(os.path.isfile('check.txt'))"
```

Comment:
Use this to test cwd of notebook by making a check.txt where I expect it to be running